sdk/src/pow/miner.rs

```diff
@@ -82,7 +80,7 @@ impl MinerBuilder {
     /// Builds the [`Miner`].
     pub fn finish(self) -> Miner {
         Miner {
-            num_workers: self.num_workers.unwrap_or(DEFAULT_NUM_WORKERS),
```

Comment:
pretty sure this isnt implemented for wasm, could you test it for wasm32-unknown-unknown and otherwise fall back to 1 on wasm?

Reply:
I was surprised either. `num_cpus` support the `wasm` targets. When wam is used, the `1`  is returned.